src/libraries/Microsoft.Extensions.Hosting/src/Resources/Strings.resx

```diff
@@ -135,6 +135,9 @@
   <data name="EnvironmentNameChangeNotSupoprted" xml:space="preserve">
     <value>The environment name changed from "{0}" to "{1}". Changing host configuration is not supported.</value>
   </data>
+  <data name="HostAbortedExceptionMessage" xml:space="preserve">
+    <value>The host was aborted prematurely.</value>
```

Comment:
@eerhardt @danmoseley Do you think this message is okay, or do you have any suggestions for improvement?

Some alternatives:
* The host was aborted early.
* The host was aborted.

Reply:
I prefer the latter:

`The host was aborted.`

"early" or "prematurely" aren't necessarily correct. In the case of the HostFactoryResolver, aborting the host is actually part of the "normal" flow. Thus, it is wasn't aborted "early" or "prematurely".